src/components/top-navigation/top-nav.css

```diff
@@ -15,3 +15,48 @@
 .main-nav svg {
    cursor: pointer;
 }
+
+.main-nav .avatar-sm {
+   border: none;
+   height: 2.15rem;
+   width: 2.15rem;
+   cursor: pointer;
+}
+
+.profile-wrapper {
+   height: 4rem;
+
+   display: flex;
```

Comment:
suggestion: remove empty line

Reply:
I tend to separate out the flex properties from the other by a line.